content/en-us/reference/engine/classes/CaptureService.yaml

```
@@ -2,8 +2,8 @@ name: CaptureService
 type: class
 category:
 memory_category: Instances
-summary: ''
-description: ''
+summary: 'A service that allows users to take screenshots of your experience.'
```

```suggestion
summary: 'A service which provides developer control over screenshot capture features.'
```

Maybe something like this, as the user is able to take screenshots without this but this will add greater developer control over the feature.